### ISSUE TITLE
Cannot tab out of Output pane with accessibility Mode on

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode#2366]
Testing #2145

Once text is selected in the output window you cannot tab out of the output pane using the TAB key.

**Comment by bpasero** [comment]
Readonly editors should always be created with the option to enable focus mode. @alexandrudima would that make sense?

**Comment by alexandrudima** [comment]
@bpasero Good idea!

**Comment by bpasero** [comment]
@alexandrudima please note my change in your land: https://github.com/Microsoft/vscode/commit/a93b4065ade5ca9256881d6c1bcc6695eb7b7cbb#diff-d327a8b1ceb558fef39f36164c644cb4R779

**Comment by alexandrudima** [comment]
@bpasero Ok, I will also put it in the internal config options computation such that all readOnly editors get it, not just the diff and output

**Comment by bpasero** [comment]
:+1:
